packages/models/organizations.coffee

```diff
@@ -1,4 +1,11 @@
 Organizations = new Mongo.Collection('organizations')
+Organizations.allow
+  insert: (userId, doc)  ->
+    false
+  update: (userId, doc, fields, modifier) ->
+    false
+  remove: (userId, doc) ->
+    false
```

Comment:
I don't think this is necessary - false should be the default, and we should just not allow client-side changes.